.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - checkout
       - run: go get -u golang.org/x/lint/golint
       - run: golint -set_exit_status $(go list ./... | grep -v /vendor/ | grep -v /explorer/idl/ | grep -v /api/idl/)
-      - run: go get -u github.com/dave/courtney
+      - run: go get -u github.com/dave/courtney@master
```

Comment:
use a specific version instead of master is perferred

Reply:
the latest is v0.3.0. i tried it and  cost more than one hour to occur timeout error when execute go.test.sh. this latest is 2018.  it may problems during processing.